{"repository":"commons-math","prod_path":"src/main/java/org/apache/commons/math/MathException.java","test_path":"src/test/java/org/apache/commons/math/MathExceptionTest.java","prod_time":"2009-09-06 23:32:50","test_time":"2009-10-08 02:56:42","add_annotation_line":0,"add_call_line":4,"add_classname_line":0,"add_condition_line":1,"add_field_line":0,"add_import_line":0,"add_packageid_line":0,"add_parameter_line":2,"add_return_line":3,"del_annotation_line":0,"del_call_line":4,"del_classname_line":0,"del_condition_line":1,"del_field_line":0,"del_import_line":0,"del_packageid_line":0,"del_parameter_line":2,"del_return_line":3,"label":"NEGATIVE","pro_Commit":"f8254ebbd3ce543854e93539eb0a3375e8e406b6","test_Commit":"f5398ad3cf1447854d7f4c720f5d255f5012531b","true_label":null}